core/src/main/java/cucumber/runtime/RuntimeOptions.java

```diff
@@ -114,11 +115,11 @@ private void parse(List<String> args) {
                 parsedGlue.add(gluePath);
             } else if (arg.equals("--tags") || arg.equals("-t")) {
                 parsedFilters.add(args.remove(0));
-            } else if (arg.equals("--plugin") || arg.equals("-p")) {
-                addPluginName(args.remove(0));
+            } else if (arg.equals("--plugin") || arg.equals("--add-plugin") || arg.equals("-p")) {
```

Comment:
Between v1.1.4 and v1.1.5 the `--glue` and `feature` options was changed from "add" to clobber (https://github.com/cucumber/cucumber-jvm/commit/5e3b0ea7e359dff9dbda4a51296c1f00d0cd109e), so this kind of semantic change has been done on already released functionality before. And isn't it more surprising for people the `--plugin` behaves differently from all other list type options?

I know the rationale for the current behaviour of `--plugin` (https://github.com/cucumber/cucumber-jvm/pull/643#issuecomment-47227892), but do think it makes thinks unnecessary hard on users that it is impossible to remove plugins defined in the `@CucumberOptions` annotation, without edit it in the source code. With this PR CucumberPro and others can use `--add-plugin` if they need the current behaviour, and the people the need the clobbering of plugins from the `@CucumberOptions` can do that too (and all list type options behave consistently).